rstar/src/algorithm/removal.rs

```diff
@@ -58,7 +58,11 @@ where
         let original_size = replace(rtree.size_mut(), 0);
 
         let m = Params::MIN_SIZE;
-        let max_depth = (original_size as f32).log(m as f32).ceil() as usize;
```

Comment:
We could replace the `m as f32` with `m.max(2) as f32` and it should fix the issue.  The value is used as an estimate for the depth of the tree as we don't store it anywhere.